### ISSUE TITLE
website broken since WordPress 5.8 update

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/genesis-srab#2]
Visiting solentroofingandbuilding.co.uk I saw a Fatal error from Gutenberg.

Steps taken to recover
- Rename Gutenberg
- That wasn't enough; I needed to deactivate SiteGround's caching in order to get the site running again.
- Jetpack widgets visibility was producing Warnings
- there were no widgets
- No images were being displayed at all on the front end.

I didn't know when this last problem occurred.  
- It could have been right back when I converted the site to WPMS!
- OR when I migrated the site to SiteGround
- The latest backup is March 21. 
- Will try on a local installation - not WPMS

Install | Theme | Works?
--------- |  --- | ------
s.b/solentro | Avada | Old theme  
s.b/srab | genesis-srab | Yes

**Comment by bobbingwide** [comment]
Try exporting from `s.b/srab`

**Comment by bobbingwide** [comment]
It would appear the site's database has been corrupt for a while. On the import to SiteGround the `$table_prefix` was changed from `wp_` to `vlz_`. 
When I imported the tables I'd made a global change to the MySQL dump file. 
This changed the data content as well as the table names. 

Some fields in the `wp_options` table got different keys, others got different values.
I have had to manually correct a number of options to resolve the problems.
